src/components/AddRestaurant/AddRestaurant.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import './AddRestaurant.css'
+
+class AddRestaurant extends Component {
+
+    render() {
+        return (
+            <button className="add-restaurant"/>
+        );
+    }
+}
```

Comment:
A component like this, which doesn't have state or any additional props or relies on `componentDidMount/Update`/whatever, can be written more simply:

```jsx
function AddRestaurant() {
  return <button className="add-restaurant" />;
}

// or

const AddRestaurant = () => <button className="add-restaurant" />;

// or even (much less common, but because it doesn't accept any props at all, and therefore doesn't have to be a function):

const AddRestaurant = <button className="add-restaurant" />;
```